cypress/support/commands.js

```diff
@@ -0,0 +1,5 @@
+Cypress.Commands.add('MovieAndvisit', (data = 'fixture:moviesApi') => {
```

Comment:
it's right approach to make actions in command

please read about `pageobject` pattern in e2e tests